Gemfile

```diff
@@ -43,6 +43,9 @@ group :system_tests do
   gem "beaker-pe",                                                               require: false
   gem "beaker-hostgenerator"
   gem "beaker-rspec"
+  gem 'rbnacl', '~> 4',                     :require => false if RUBY_VERSION >= '2.2.6'
```

Comment:
What are you using these Gems for? The Gemfile will be overwritten by a pdk run so these changes would eventually get removed. If you want them permanent they'll have to be moved to .sync.yml.

Reply:
The commit already has some information:
https://github.com/puppetlabs/puppetlabs-apache/pull/1809/commits/7703346bc27e5d264204a429a2a37041b6d4907d

Reply:
Thanks! We will look in to adding these to puppet-module-gems.

Reply:
We changed it to allow newer version of rbnacl as well: https://github.com/voxpupuli/modulesync_config/commit/58f32bf08917444081c84ffb9b58783dda4f31e6

Reply:
Can you remove these new Gems for merge? We will be adding them to the puppet-module-gems in due time and these would conflict.